src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -859,6 +911,99 @@ private static void OnPingComplete(object sender, PingCompletedEventArgs e)
             ((TestConnectionCommand)e.UserState)._pingComplete.Set();
         }
 
+        /// <summary>
+        /// The class contains information about the TCP connection test
+        /// </summary>
+        public class TcpTestStatus
+        {
+            /// <summary>
+            /// Initializes a new instance of the <see cref="TcpTestStatus"/> class.
+            /// </summary>
+            /// <param name="testNum">The number of this test.</param>
+            /// <param name="source">The source machine name or IP of the test.</param>
+            /// <param name="destination">The destination machine name or IP of the test.</param>
+            /// <param name="destinationAddress">The resolved IP from the destination</param>
+            /// <param name="port">The port used for the connection.</param>
+            /// <param name="latency">The latency of the test.</param>
+            /// <param name="result">The result of the test.</param>
+            internal TcpTestStatus(int testNum, string source, string destination, string destinationAddress, int port, long latency, TcpConnectionTestResult result)
+            {
+                TestNum = testNum;
+                Source = source;
+                Destination = destination;
+                DestinationAddress = destinationAddress;
+                Port = port;
+                Latency = latency;
+                Result = result;
+            }
+
+            /// <summary>
+            /// Gets and sets the count of the test.
+            /// </summary>
+            public int TestNum { get; set; }
```

Comment:
I think this should either just be `Test` or be fully named `TestNumber` -- also, the formatting header name should probably reflect the real property value (or an least a property that is accessible to the user) rather than a short name that may lead to confusion. (Name in formatter should match up to a property users can use, if that makes sense?)

Reply:
Please don't use "test" in the name.

Reply:
What do you guys think about `ConnectionNum` for the property name? I'm having a tough time coming up with a property name that is not overly descriptive, and not too ambiguous 🤔

I totally agree it makes sense to have the name of the property line up with the header. I may have overlooked that one 😅

Reply:
I would err on the side of keeping it as short as possible. But that might be mainly a preference thing for me; it feels very wasteful to have a lengthy column header when we know that our data will only fill a couple characters of space here.

That's why my initial thought was to use `Test`, but perhaps something else is needed here. Perhaps `Attempt`? Still a _little_ long, but not overly so...

Reply:
I think `Attempt` would work in this case, it's a pretty good balance! I'll make those changes!

Reply:
Perhaps "Id" will be more appropriated.

Reply:
Ah that's another good option! Do you reckon it may be a bit misleading, or intuitive enough that it sort of explains itself?

Reply:
I think it is used very often, specially in database area.

Reply:
That's a good point, it's short and sweet! Upon trying it out, `Attempt` seems a bit long. I'll make that change 😊

Reply:
Updated! Thanks 🙂